tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -84,14 +88,177 @@ class MatchMulSigmoid : public RewritePattern {
   OperationName sigmoid_name_;
 };
 
+static FailureOr<TFOp> CreateContractionWithBiasAddOp(PatternRewriter &rewriter,
+                                                      OpPropertyHelper &helper,
+                                                      Operation *contraction_op,
+                                                      Operation *bias_add_op) {
+  // Get all operands for fused op
+  Value input = contraction_op->getOperand(0);
+  Value filter = contraction_op->getOperand(1);
+  Value bias = bias_add_op->getOperand(1);
+
+  SmallVector<Value> operands;
+  operands.push_back(input);
+  operands.push_back(filter);
+  operands.push_back(bias);
+
+  // Get the contraction type, MatMul/Conv2D/DepthwiseConv2dNative etc.
+  std::string fused_contraction_type;
```

Comment:
I may suggest a name like `fused_constraction_op_name`. A name with type may give the impression that it's a `Type`